### PR TITLE
Configure the driver

### PR DESCRIPTION
**Pull request by Fedosin**
This commit creates a static configmap with cinder config that contains the link to clouds.yaml file, generated by CCO.

### CODE REVIEW

File: assets/node.yaml [hunk 1]
```diff
@@ -39,7 +39,7 @@ spec:
             - name: CSI_ENDPOINT
               value: unix://csi/csi.sock
             - name: CLOUD_CONFIG
-              value: /etc/kubernetes/clouds.conf
+              value: value: /etc/kubernetes/config/cloud.conf
```
Comment (EmilienM) [inline assets/node.yaml:42]: typo, you have two times "value:"

Comment (Fedosin) [reply assets/node.yaml:42]: yeah, thank you!

File: manifests/07_deployment.yaml [hunk 2]
```diff
@@ -59,15 +56,3 @@ spec:
...
-        # Extract ca-bundle.pem to /usr/share/pki/ca-trust-source if present.
-        # Let the pod start when the ConfigMap does not exist or the certificate
-        # is not preset there. The certificate file will be created once the
-        # ConfigMap is created / the cerificate is added to it.
-        configMap:
-          name: cloud-provider-config
-          items:
-            - key: ca-bundle.pem
-              path: ca-bundle.pem
-          optional: true
```
Comment (jsafrane) [inline manifests/07_deployment.yaml:73]: From commit message:

>  if we try to mount the cacert volume, it fails.

How can it fail if it's `optional: true`?

Comment (Fedosin) [reply manifests/07_deployment.yaml:73]: The error message is

> The Deployment "openstack-cinder-csi-driver-operator" is invalid: spec.template.spec.containers[0].volumeMounts[0].name: Not found: "cacert"

**Pull request merged**